src/commands/local.ts

```diff
@@ -499,8 +545,10 @@ async function listenForChanges(sdkPathRelative: any | undefined) {
             }
           }
 
-          console.clear();
-          log.info("\x1b[36m%s\x1b[0m", "Change detected, reloading...");
+          if (showLogs) {
```

Comment:
Instead of adding a parameter that changes the behavior of the class, I think it would be more clear and future-proof to move this console clear and log outside of the function.